index.html

```diff
@@ -0,0 +1,17 @@
+
+<!DOCTYPE html>
+<!-- A non-handlebars version of search.html -->
```

Comment:
this is super useful!  in the future, I'd like to better understand how our current handlebars setup works, or migrate to some other templating / markup tool that I understand better (like JSX or plain HTML).  

Glad to see this option for bypassing the handlebars bits for local blockbuilder-search development on the react app part of the blockbuilder-search UI.